conf/messages

```diff
@@ -969,4 +956,12 @@ returns.list.noTaxDue=You did not owe any tax for this return
 returns.list.leftToPay=Total left to pay
 returns.list.taxOwed=Tax owed for return
 returns.list.sentDate=Sent to HMRC on {0}
-returns.paymentTakes3Days=Payments may take up to 3 working days to be shown on this page
\ No newline at end of file
+returns.paymentTakes3Days=Payments may take up to 3 working days to be shown on this page
+
+multiple-disposals.howManyProperties.title=How many disposals are you including in this return?
+multiple-disposals.howManyProperties.p1=The property disposals must all have the same 'completion' date. The contract 'exchange' dates must all be in the same tax year.
```

Comment:
we use curly quotes like the ones here https://smartquotesforsmartpeople.com/ instead of straight quotes. You can copy and paste the right and left quotes from that page